modules/perception/camera/app/cipv_camera.cc

```diff
@@ -190,12 +202,12 @@ bool Cipv::MakeVirtualEgoLaneFromYawRate(const float yaw_rate,
   right_lane_line->line_point.clear();
 
   if (b_image_based_cipv_ == false) {
-    for (uint32_t i = 0; i < 120; i += 5) {
+    for (uint32_t i = 1; i < 25; i += 1) {
```

Comment:
Could it be a const global variable to better describe the magic number `25`?

Reply:
25 is changed to constant, kMaxNumVirtualLanePoint, which doesn't have to be configurable.